examples/gke-a3-megagpu/gke-a3-megagpu.yaml

```diff
@@ -50,6 +50,15 @@ vars:
   accelerator_type: nvidia-h100-mega-80gb
   version_prefix: "1.32."
 
+  enable_periodic_health_checks: false # Make this true to run CHS (healthchecks)
+  health_check_schedule: "0 0 * * *" # Run the health check daily by default
```

Comment:
What's the period mentioned here? As discussed, let's run it weekly on sundays by default. Also, expose this variable in the [deployment file](https://github.com/GoogleCloudPlatform/cluster-toolkit/blob/develop/examples/gke-a3-megagpu/gke-a3-megagpu-deployment.yaml) in the folder.